.github/workflows/trivy.yml

```diff
@@ -6,37 +6,8 @@ on:
     - cron:  '0 8 * * *'
   workflow_dispatch:
 jobs:
-  trivy_scanner:
-    name: Trivy scanner for docker
-    runs-on: ubuntu-20.04
-    steps:
-
-      - name: newrelic/infrastructure
-        uses: aquasecurity/trivy-action@master
-        with:
-          image-ref: 'docker.io/newrelic/infrastructure:latest'
-          format: 'table'
-          exit-code: '1'
-          ignore-unfixed: true
-          vuln-type: 'os,library'
-          severity: 'CRITICAL,HIGH'
-
-      - name: newrelic/k8s-events-forwarder
-        uses: aquasecurity/trivy-action@master
-        with:
-          image-ref: 'docker.io/newrelic/k8s-events-forwarder:latest'
-          format: 'table'
-          exit-code: '1'
-          ignore-unfixed: true
-          vuln-type: 'os,library'
-          severity: 'CRITICAL,HIGH'
-
-      - name: newrelic/nri-forwarder
-        uses: aquasecurity/trivy-action@master
-        with:
-          image-ref: 'docker.io/newrelic/nri-forwarder:latest'
-          format: 'table'
-          exit-code: '1'
-          ignore-unfixed: true
-          vuln-type: 'os,library'
-          severity: 'CRITICAL,HIGH'
\ No newline at end of file
+  docker-trivy:
+    uses: newrelic/infrastructure-agent/.github/workflows/component_trivy.yml@master
```

Comment:
I think you can now skip the repo name and the `@master` thing, which may be helpful for forks or when making changes to the referenced file :)

https://github.blog/changelog/2022-01-25-github-actions-reusable-workflows-can-be-referenced-locally/

Reply:
We will have another task to review all components and remove repo binding.